app/models/hmis/build_report.rb

```diff
@@ -0,0 +1,66 @@
+module Hmis
+  class BuildReport
+    def run!
+      # we just flush the whole damned thing every time
+      Hmis::Report.delete_all
+      at = MatchDecisions::Base.arel_table
```

Comment:
I have a new pattern for these, not completely necessary, but I'm liking md_t (MatchDecision_Table) instead of the hard to determine which arel_table we're dealing with `at`.